capellambse/extensions/reqif/elements.py

```diff
@@ -279,6 +279,9 @@ def __repr__(self) -> str:  # pragma: no cover
 
         return f'<{mytype} {"/".join(reversed(path))!r} ({self.uuid})>'
 
+    def _short_repr_(self, name: str = "") -> str:
+        return super()._short_repr_(name or self.name or self.long_name)
```

Comment:
Why not just `return f"<{type(self).__name__} {self.name or self.long_name!r} ({self.uuid})>"` here? No need to overcomplicate the superclass to accomodate for an odd subclass. That's what inheritance is about after all. And accepting a `name` from externally into `_short_repr_` feels just ... wrong. It's not an internal function, so the universe can call it - we really need to take better care of our public API.

Reply:
The `super()._short_repr_(name or self.name or self.long_name)` does exactly what you ask for. Why not reuse this from the super class? I thought that is what inheritance is about. It is also dry.
Am I misunderstanding that `_short_repr_` is an internal function which is only ever called by `__repr__` from `GenericElement` or `ElementList`. Never ever, by the user directly. That's why `repr()` exists.